crates/turborepo-lib/src/config/env.rs

```diff
@@ -63,7 +63,7 @@ impl EnvVars {
 impl ResolvedConfigurationOptions for EnvVars {
     fn get_configuration_options(
         &self,
-        _existing_config: &ConfigurationOptions,
+        _existing_config: Option<&ConfigurationOptions>,
```

Comment:
This was made an option because in all the places where `ResolvedConfigurationOptions` is used, this argument is only provided once.  That left us constructing and passing `ConfigurationOptions::default()` everywhere, which was hard to follow (considering it's almost never used).

Reply:
all good - reverted this change in 0e9eaffff540fa0fc5fd9cec65b8f42cea6d9dc1

I just found it confusion.  Options that aren't optional, and all that.  by "provided once" I meant "passed in once".

But all good - I trust your instincts here!